api/documentation/controller.py

```diff
@@ -696,6 +696,48 @@ def addSchemes(self):
             }
         )
 
+        self.addComponent(
+            'schema', 'EditWorkPostForm', 'object',
+            {
+                'title': {'type': 'string'},
+                'subtitle': {'type': 'string'},
+                'contributor-role': {
+                    'type': 'array',
+                    'items': {'type': 'string'}
+                },
+                'contributor-name': {
+                    'type': 'array',
+                    'items': {'type': 'string'}
+                },
+                'series': {'type': 'string'},
+                'series_position': {'type': 'integer'},
+                'medium': {'type': 'string'},
+                'language': {'type': 'string'},
+                'publisher': {'type': 'string'},
+                'imprint': {'type': 'string'},
+                'issued': {
+                    'type': 'string',
+                    'format': 'date'
+                },
+                'rating': {'type': 'integer'},
+                'summary': {'type': 'string'}
+            }
+        )
+
+        self.addComponent(
+            'schema', 'ChangeBookCoverForm', 'object',
+            {
+                'rights_status': {'type': 'string'},
+                'rights_explanation': {'type': 'string'},
+                'cover_file': {
+                    'type': 'string',
+                    'format': 'binary'
+                },
+                'cover_url': {'type': 'string'},
```

Comment:
Does the endpoint expect one or the other of these? I don't think we can capture that as a requirement, but a comment along those lines could help clarify things!

(Or if that assumption is incorrect, what the functionality of both fields are)

Reply:
Comment added.